lib/BiochemistryAPI/utils.py

```diff
@@ -93,44 +95,61 @@ def _get_mol(structure):
     return mol
 
 
-def make_mol_tuples(compound_dict, id_key="id", struct_key='structure', struct_type='inchi'):
+def make_mol_tuples(
+    compound_dict, id_key="id", struct_key="structure", struct_type="inchi"
+):
     """
-    Creates named tuples with (compound_id, RDKit Mol Object) from a dict with SMILES or InChI
+    Creates named tuples with (compound_id, RDKit Mol Object)
+     from a dict with SMILES or InChI
     """
     MolTuple = namedtuple("MolTuple", "id mol maccs_fp rdkit_fp")
     tups = []
     for comp in compound_dict:
         mol = _get_mol(comp[struct_key])
         if mol:
-            tups.append(MolTuple(comp[id_key],
-                                 mol,
-                                 AllChem.GetMACCSKeysFingerprint(mol),
-                                 AllChem.RDKFingerprint(mol)))
+            tups.append(
+                MolTuple(
+                    comp[id_key],
+                    mol,
+                    AllChem.GetMACCSKeysFingerprint(mol),
+                    AllChem.RDKFingerprint(mol),
+                )
+            )
     return tups
 
 
 def substructure_search(query, structures):
     """Performs substructure search on 'query' in 'structures'"""
     pattern = AllChem.MolFromSmarts(query)
-    return [x.id for x in structures if x.mol and x.mol.HasSubstructMatch(pattern)]
+    return [x.id for x in structures
+            if x.mol and x.mol.HasSubstructMatch(pattern)]
 
 
-def similarity_search(query, structures, fp_type='maccs', min_similarity=0.8):
-    """Perform return compound ids where tanimoto similarity of fingerprint 'fp_type is greater
-    than 'min_similarity'"""
+def similarity_search(query, structures, fp_type="maccs", min_similarity=0.8):
+    """Perform return compound ids where tanimoto similarity of fingerprint
+     'fp_type is greater than 'min_similarity'"""
     if not isinstance(min_similarity, float) or not 0 <= min_similarity <= 1.0:
```

Comment:
If you wanted to be a HUGE nerd you could split this into a `TypeError` and a `ValueError`